db/seeds.rb

```diff
@@ -8,3 +8,4 @@
 require File.dirname(__FILE__) + '/seed_helper.rb'
 
 create_roles_and_permissions
+populate_languages
```

Comment:
Looking at the cap tasks, it looks like this will only be called when the cap seed task is called. This also deletes any existing roles/languages, so perhaps instead it should check to see if the corresponding role/language record already exist in the DB and create it if it doesn't (otherwise it's going to bork any existing relations)? We could then add the seed task as a subtask of the deploy task.